Themes/Everblush/README.md

```diff
@@ -30,58 +35,58 @@ To import the theme manually, follow these steps:
 ```json
 {
   "controlStyles[0].target": "Border#AcrylicBorder",
-  "controlStyles[0].styles[0]": "Background=#141b1e",
-  "controlStyles[0].styles[1]": "BorderBrush=#268ccf7e",
+  "controlStyles[0].styles[0]": "Background:=#141b1e",
+  "controlStyles[0].styles[1]": "BorderBrush:=#268ccf7e",
   "controlStyles[1].target": "Border#AcrylicOverlay",
-  "controlStyles[1].styles[0]": "Background=#141b1e",
+  "controlStyles[1].styles[0]": "Background:=#141b1e",
   "controlStyles[2].target": "StartDocked.SearchBoxToggleButton > Grid > Border",
   "controlStyles[2].styles[0]": "Background=#232a2d",
   "controlStyles[2].styles[1]": "BorderBrush=transparent",
   "controlStyles[3].target": "StartMenu.ExpandedFolderList > Grid > Border",
   "controlStyles[3].styles[0]": "Background=#232a2d",
   "controlStyles[4].target": "TextBlock#PlaceholderText",
-  "controlStyles[4].styles[0]": "Foreground=#80b3b9b8",
+  "controlStyles[4].styles[0]": "Foreground:=#80b3b9b8",
   "controlStyles[5].target": "Windows.UI.Xaml.Controls.Button",
-  "controlStyles[5].styles[0]": "Background=#d28ccf7e",
+  "controlStyles[5].styles[0]": "Background:=#d28ccf7e",
   "controlStyles[6].target": "StackPanel > Windows.UI.Xaml.Controls.Button",
-  "controlStyles[6].styles[0]": "Background=transparent",
-  "controlStyles[6].styles[1]": "BorderBrush=transparent",
+  "controlStyles[6].styles[0]": "Background:=transparent",
+  "controlStyles[6].styles[1]": "BorderBrush:=transparent",
   "controlStyles[7].target": "Microsoft.UI.Xaml.Controls.ItemsRepeater > Windows.UI.Xaml.Controls.Button",
-  "controlStyles[7].styles[0]": "Background=transparent",
-  "controlStyles[7].styles[1]": "BorderBrush=transparent",
+  "controlStyles[7].styles[0]": "Background:=transparent",
+  "controlStyles[7].styles[1]": "BorderBrush:=transparent",
   "controlStyles[8].target": "TextBlock#DisplayName",
-  "controlStyles[8].styles[0]": "Foreground=#b3b9b8",
+  "controlStyles[8].styles[0]": "Foreground:=#e5c76b",
```

Comment:
I replaced `:=` with `=` in most places since `:=` is unnecessary for non-XAML property values. Any reason you prefer having `:=`?